### PR TITLE
add download2.camlcity.org mirrors

### PR DESCRIPTION
**Pull request by fdopen**
download.camlcity.org is down far too often ...

### CODE REVIEW

File: packages/ocamlfind/ocamlfind.1.5.6/url [hunk 1]
```diff
@@ -1,2 +1,3 @@
 archive: "http://download.camlcity.org/download/findlib-1.5.6.tar.gz"
 checksum: "91585dd5459cb69bfd9a0689bf222403"
+mirrors: [ "http://download2.camlcity.org/download/findlib-1.5.7.tar.gz" ]
```
Comment (hcarty) [inline packages/ocamlfind/ocamlfind.1.5.6/url]: Mismatch here - `1.5.7` in the mirror, should be probably be `1.5.6`.

Comment (fdopen) [reply packages/ocamlfind/ocamlfind.1.5.6/url]: Indeed. Fixed by a forced push.

Comment (hcarty) [reply packages/ocamlfind/ocamlfind.1.5.6/url]: Thanks!

**Comment by hcarty** [comment]
I think this is OK to merge as it only affects opam's fallback if the main download site is down.  I'd like another reviewer to approve (and/or merge) though, out of a general sense of paranoia.

**Comment by hannesm** [comment]
thanks, lgtm

**Pull request merged**